features/collection-order-filter.feature

```diff
@@ -0,0 +1,411 @@
+Feature: Order filter on collections
+  In order to retrieve ordered large collections of resources
+  As a client software developer
+  I need to retrieve collections ordered properties
+
+  @createSchema
+  Scenario: Get collection ordered in ascending order on an integer property and on which order filter has been enabled in whitelist mode
+    Given there is "30" dummy objects
+    When I send a "GET" request to "/dummies?order[id]=asc"
+    Then the response status code should be 200
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/ld+json"
+    And the JSON should be equal to:
+    """
+    {
+      "@context": "/contexts/Dummy",
+      "@id": "/dummies?order[id]=asc",
+      "@type": "hydra:PagedCollection",
+      "hydra:nextPage": "/dummies?order%5Bid%5D=asc&page=2",
```

Comment:
Btw could not change this line to `"hydra:nextPage": "/dummies?order[id]=asc&page=2",` :(